tests/integration/models/tags.ts

```diff
@@ -3,6 +3,8 @@ import * as _ from 'lodash';
 import { expect } from 'chai';
```

Comment:
I might miss something, why do we need tags change to allow getting the release by raw_version ?

Reply:
It's not just getting a release by raw_version that is added by this PR, it's *referencing* a release by raw_version in general. So for example we can now finalize and invalidate releases by raw_version as well. Basically anything that would expect to be able to reference a release by id/commit should also be able to do so by raw_version. This includes tags (add, remove, update, etc.)